docs/source/add_new_model.mdx

```diff
@@ -793,9 +793,21 @@ You have now finished the coding part, congratulation! 🎉 You are Awesome! 
 **12. Upload the models to the model hub**
 
 In this final part, you should convert and upload all checkpoints to the model hub and add a model card for each
-uploaded model checkpoint. You should work alongside the Hugging Face team here to decide on a fitting name for each
+uploaded model checkpoint. You can get familiar with the hub functionalities by reading at our [Model sharing and uploading Page](https://huggingface.co/docs/transformers/model_sharing). You should work alongside the Hugging Face team here to decide on a fitting name for each
 checkpoint and to get the required access rights to be able to upload the model under the author's organization of
-*brand_new_bert*.
+*brand_new_bert*. The `push_to_hub` method, present in all models in `transformers`, is a quick and efficient way to push your checkpoint to the hub. A little snippet is pasted below:
+
+
+```python
+from pathlib import Path
+
+brand_new_bert.push_to_hub(
+    repo_path_or_name=Path("/path/to/checkpoint/"),
+    organization="<YOUR_USERNAME>",
```

Comment:
This should be removed. `organization` should be used to push to an organization, not in the user's namespace.